### PR TITLE
 Checks if string is palindrome

### PR DESCRIPTION
**Pull request by devfalah**
Checks if string is palindrome 
```dart
print('flutterrettulf'.isPalindrom);//true
print('20011001'.isPalindrom,);//false
```

### CODE REVIEW

File: readable/lib/src/extensions/string/string.dart [hunk 1]
```diff
@@ -2,4 +2,7 @@
 extension StringReadableX on String {
   ///convert  String to List of Characters
   List<String> toChars() => split('');
+
+  /// Checks if string is Palindrom.
+  bool get isPalindrom => split('').reversed.join() == this;
```
Comment (maxzod) [inline readable/lib/src/extensions/string/string.dart:7]: Why split it ?
You can use  reversed == this

**Review by maxzod — CHANGES_REQUESTED** [review]

File: readable/lib/src/extensions/string/string.dart (hunk 1 above)
Comment (devfalah) [reply readable/lib/src/extensions/string/string.dart:7]: because The getter 'reversed' isn't defined for the type 'String'.

Comment (maxzod) [reply readable/lib/src/extensions/string/string.dart:7]: Yes , you are right 😅

**Review by maxzod — APPROVED** [review]

File: readable/lib/src/extensions/string/string.dart (hunk 1 above)
Comment (devfalah) [reply readable/lib/src/extensions/string/string.dart:7]: do I write reversed getter in StringReadableX and use it in isPalindrom ?

**Pull request merged**